source/core.py

```diff
@@ -597,6 +597,9 @@ def main():
 	log.debug("Initializing tones")
 	import tones
 	tones.initialize()
+	log.debug("Initializing sound split")
+	import audio
+	audio.initialize()

```

Comment:
`audio` should be reinitialized when user restores config both to the last saved, and to the factory defaults in `core.resetConfiguration`.